satchless/pricing/__init__.py

```diff
@@ -47,7 +29,39 @@ def __eq__(self, other):
         return False
 
     def __ne__(self, other):
-        return not self == other
+        return not (self == other)
+
+    def __gt__(self, other):
```

Comment:
Would the `functools.total_ordering` class decorator be able to reduce the number of protocol methods requiring definition?